test-app-reactive/src/main/resources/application-localstack.yml

```diff
@@ -2,18 +2,47 @@ hmpps.sqs:
   provider: localstack
   queues:
     inboundqueue:
-      queueName: async-inbound-queue
```

Comment:
service failed to start locally until I added in all the queues that are now in use by the tests